src/Data/Scope.elm

```diff
@@ -70,6 +74,9 @@ toLabel scope =
         Textile ->
             "Textile"
 
+        Veli ->
+            "Véhicule intermédiaire"
```

Comment:
What about "Véhicules intermédiaires" to be consistent with "Objets"?

Reply:
Got it.